.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,30 @@
+name: PyPI Publish
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  pypi-publish:
```

Comment:
The job doesn't rely on test, it's probably a good idea to set it up so it requires all tests to pass before publishing.